devices/ikea.js

```diff
@@ -90,6 +90,13 @@ module.exports = [
         description: 'TRADFRI LED bulb E27 1000 lumen, dimmable, opal white',
         extend: tradfriExtend.light_onoff_brightness(),
     },
+    {
+        zigbeeModel: ['TRADFRIbulbG125E27WSopal470lm'],
+        model: 'LED1936G5',
+        vendor: 'IKEA',
+        description: 'TRADFRI LED globe-bulb E27 470 lumen, dimmable, white spectrum, opal white',
+        extend: tradfriExtend.light_onoff_brightness_colortemp(),
```

Comment:
Please add the color temp range as explained here: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
got this back cluster 'lightingColorCtrl', data '{"colorTempPhysicalMax":454,"colorTempPhysicalMin":250}' which matched the colorTempRange of the tradfriExtend funkction. So i didn't added it, because it was already there.

Reply:
Btw. based on the productdescription for the LED1937T5, that bulb has also the colortemp feature.

Reply:
Ah sorry, missed that it has already been defined in `tradfriExtend`, thanks!